OpenLIFULib/OpenLIFULib/transducer.py

```diff
@@ -159,3 +160,19 @@ def observe_transform_modified(self, callback : "Callable[[SlicerOpenLIFUTransdu
 
     def stop_observing_transform_modified(self, tag:int) -> None:
         self.transform_node.RemoveObserver(tag)
+
+    def set_current_transform_to_match_transform_node(self, transform_node : vtkMRMLTransformNode) -> None:
+        """Set the matrix on the current transform node of this transducer to match the matrix of a given transform node.
+        (This is done by a copy not reference, so it's a one-time update -- the tranforms do not become linked in any way.)"""
+
+        transform_matrix = vtk.vtkMatrix4x4()
+        transform_node.GetMatrixTransformToParent(transform_matrix)
+        self.transform_node.SetMatrixTransformToParent(transform_matrix)
+
+    def move_node_into_transducer_sh_folder(self, node : vtkMRMLNode) -> None:
+        """In the subject hiearchy, move the given `node` into this transducer's transform node folder."""
+        shNode = slicer.vtkMRMLSubjectHierarchyNode.GetSubjectHierarchyNode(slicer.mrmlScene)
+        shNode.SetItemParent(
+            shNode.GetItemByDataNode(node),
+            shNode.GetItemParent(shNode.GetItemByDataNode(self.transform_node)),
+        )
```

Comment:
@sadhana-r
added this function under SlicerOpenLIFUTransducer to make it simpler to move things into the transducer folder